### PR TITLE
Mostly Non-Latin characters

### PR DESCRIPTION
**Pull request by nsmsn**
I tagged the Khmer script fonts with "Khmer script" and tagged Thai fonts as "looped" or "loopless". 

Updated a few Western faces missing tags as well

### CODE REVIEW

**Comment by katydecorah** [comment]
@nsmsn thanks for contributing 💜! Can you tell me if the tag you're adding "Khmer script" is different than the subset "Khmer" (which is already available as a filter)? This is beyond my knowledge, so I want to make sure that they aren't the same thing.

**Comment by davelab6** [comment]
The looped/loopless distinction isn't made anywhere in the GF system, but
the scripts subsets could also be tags

**Comment by nsmsn** [comment]
"Khmer script" refers to the alphabet, so in that way its similar to the Thai or Telugu subsets. 
[Khmer Alphabet Wikipedia](https://en.wikipedia.org/wiki/Khmer_alphabet)

I can modify those tags and re-submit. I also noticed a new tag I should re-label as "high contrast" instead of "contrast"

**Comment by nsmsn** [comment]
Let's try this again. I removed all "Khmer script" tags. 

Looped/loopless isn't specified in Google Fonts, but the fonts by Cadson Demak make this distinction.

File: families.json [hunk 1]
```diff
@@ -740,14 +740,14 @@
   {"family": "Supermercado One", "tags": ["geometric", "swash", "low contrast"]},
   {"family": "Sura", "tags": []},
   {"family": "Suranna", "tags": ["high contrast"]},
-  {"family": "Suravaram", "tags": []},
-  {"family": "Suwannaphum", "tags": []},
+  {"family": "Suravaram", "tags": ["brush", "script," "handwritten"]},
```
Comment (katydecorah) [inline families.json]: @nsmsn small syntax error here, misplaced comma 🙈. should be:

```
["brush", "script", "handwritten"]
```

File: families.json [hunk 2]
```diff
@@ -378,15 +378,15 @@
   {"family": "Kadwa", "tags": []},
   {"family": "Kalam", "tags": ["handwritten"]},
   {"family": "Kameron", "tags": ["slab", "egyptian"]},
-  {"family": "Kanit", "tags": ["humanist", "futuristic"]},
-  {"family": "Kantumruy", "tags": []},
+  {"family": "Kanit", "tags": ["humanist", "futuristic", "loopless Thai"]},
```
Comment (katydecorah) [inline families.json]: @nsmsn can "loopless Thai" just be "loopless"? Or is there a strong distinction that you are making here?

Comment (nsmsn) [reply families.json]: yes, "loopless" should work here. My bad

**Comment by nsmsn** [comment]
Fixed the comma on line 743.
Thai fonts are labeled "looped" and "loopless" now.

**Comment by katydecorah** [comment]
@nsmsn awesome! Thanks so much for making those edits and contributing 💟

**Comment by nsmsn** [comment]
Thanks for hanging in there with me. Keep up the great work!

**Pull request merged**